src/main.tsx

```diff
@@ -14,9 +15,11 @@ root.render(
     <ApolloProvider>
       <ThemeProvider>
         <NotificationsProvider limit={3}>
-          <AuthProvider fallback={<FullPageLoader />}>
-            <Routes />
-          </AuthProvider>
+          <ContextProvider>
```

Comment:
I wouldn't say I like this idea. It kind of means contexts are global states.

Reply:
agree